aws/rakefiles/deploy.rake

```diff
@@ -154,27 +154,24 @@ task :install_charts => [:configure_kubectl, :generate_modules, :setup_system_co
   installed_charts = `helm list -q -a`
   installed_charts = installed_charts.split("\n")
   @gpii_helmcharts.each do |chart, attrs|
-    # Extracting release name and namespace
-    if attrs && attrs['release_name']
-      chart_name = attrs['release_name']
-    else
-      chart_name = chart.match(/^\d+\-(.*)/)[1]
-    end
-    if attrs && attrs['release_namespace']
-      chart_namespace = attrs['release_namespace']
-    else
-      chart_namespace = 'default'
-    end
-
+    # Extracting release attributes
+    attrs ||= {}
+    chart_name = attrs.fetch('release_name', chart)
```

Comment:
The regex removing the leading digits is no longer needed?

Reply:
Yes, we no longer naming charts with numerical prefixes, since we introduced yaml charts config.